src/Makefile.am

```diff
@@ -0,0 +1,11 @@
+bin_PROGRAMS = snap-run
+snap_run_SOURCES = main.c seccomp_utils.c utils.c
+snap_run_CFLAGS = \
```

Comment:
check syntax restored (for shell scripts), looking at .c and installing the small udev helper